### DISCUSSION TITLE
I am facing an error React Native error

### DISCUSSION THREAD

**Discussion by CharmingDev1219** [community/community#35603] [Apps, API and Webhooks]
# Environment
Ubuntu 18.2.4

#Description
```bash
npx react native run-android
```
I am facing some errors when I run this.

Here is error message

```
error: failed to load configration
```

Please help me to fix this error if you have some experience.

Thanks.

**Comment by tccpro** [comment]
I had same bug once, but try this one.

```bash
yarn global remove expo
yarn global remove expo-cli
npm uninstall -g expo
npm uninstall -g expo-cli
```
And also, please clear `node_modules` and `package.json`, `yarn.lock` in your app folder.

Hope will be helpful.

Thanks.

**Reply by CharmingDev1219** [reply]
But still getting same error

Would you please let me know in more detail?

Thanks.

**Reply by tccpro** [reply]
I see  you are using `Ubuntu` OS. I fixed error on `Windows 10` OS.

You may need to set `git-repository` so that you can fetch from git.

**Reply by CharmingDev1219** [reply]
Thank you for your quick reply.
But I have installed already.
:(

**Reply by tccpro** [reply]
Then please try this one.
`npm cache clean --force`

**Reply by CharmingDev1219** [reply]
Thank you it does work,
I have fixed.
Thank you for your support!
